### PR TITLE
fix: use selfsigned instead of devcert for certificate generation

### LINKED ISSUE #1649: Devcert require sudo rights to work correct. Need to be replaced more suitable self-signed certificate generation tool
Some operation of devcert require sudo rights, which are not available from docker. This is causing issues to start SK in case missing security.json.

### PR DESCRIPTION
**Pull request by KEGustafsson**
Fixes #1649 seen with Devcert self-signed certificate generation tool. Devcert require sudo for certain operations and that is not available from docker.

### CODE REVIEW

**Comment by tkurki** [comment]
@cmotelet how does this look to you? Will this work on windows?

**Comment by KEGustafsson** [comment]
I tested selfsigned image on Windows 10 with Docker Desktop. Docker-compose file applied from Cmd and WSL2. SK contsiner came up fine.

File: package.json [hunk 1]
```diff
@@ -116,7 +115,9 @@
     "ms": "^2.1.2",
     "ncp": "^2.0.0",
     "node-fetch": "^2.6.0",
+    "node-forge": "^1.3.1",
```
Comment (tkurki) [inline package.json]: Is this needed? It is not referenced from js and I don't see any reference to it in `selfsigned` docs.

Comment (KEGustafsson) [reply package.json]: Selfsigned was updated recently and these missing dependecies are now included to package.

**Comment by cmotelet** [comment]
> @cmotelet how does this look to you? Will this work on windows?

@tkurki , Sorry I missed this discussion !
Since August 25, 2023 in the Windows installer version 1.1.0, I automatically generate a self-signed certificate valid for 10 years.
https://github.com/SignalK/signalk-server-windows/blob/master/NSIS/signalk-server.nsi#L359
I've just validated version 2.4.1 of the Signal K server and the new server-integrated generation with `selfsign` of the certificate also works perfectly under Windows.

**Pull request merged**